### ISSUE TITLE
预训练代码

### ISSUE THREAD

**Issue by liweitj47** [Ethan-yt/guwenbert#15]
请问方便公布一下预训练的代码吗？

**Comment by Ethan-yt** [comment]
我们预训练使用的fairseq命令行工具，只需要一行命令，所以没有预训练代码。
如果想要预训练，可以尝试fairseq，也可以尝试huggingface提供的代码。

**Comment by Shenxin0925** [comment]
您好，请问方便分享下这行命令涉及哪些超参数吗?我想向您学习一下重构词表的预训练方法。

**Comment by Ethan-yt** [comment]
readme里面已经写了使用的超参数

Name | Value
-- | --
Batch size | 2,048
Seq Length | 512
Optimizer | Adam
Learning Rate | 2e-4(base), 1e-4 (large)
Adam-eps | 1e-6
Weight Decay | 0.01
Warmup | 5K steps, linear decay of learning rate after.
